routes/controller.go

```diff
@@ -2,6 +2,8 @@ package routes
 
 import "gorm.io/gorm"
 
+// Controller for the whole API, all endpoints should be implemented on this struct.
+// This allows to have a single point of access to the database and other services.
 type Controller struct {
 	DB *gorm.DB
 }
```

Comment:
More than a type, isn't there a way to create an interface with a "configureRoutes" function ?

Reply:
That could be an option, I'll investigate it tomorrow.